packages/core/src/modules/attendance/attendee-service.ts

```diff
@@ -114,21 +114,55 @@ export class AttendeeServiceImpl implements AttendeeService {
 
     const registerTime = new Date()
 
+    const displayName = user.firstName && user.lastName ? `${user.firstName} ${user.lastName}`.trim() : user.email
```

Comment:
This should probably be extracted into a function inside the users file in packages/types